draft-secure-credential-transfer.md

```diff
@@ -538,6 +538,42 @@ Unauthorized - calling device is not authorized to read the secure content of th
 Not Found - mailbox with provided mailboxIdentifier not found.
 
 
+## RelinquishMailbox
+
+An application running on a remote device can invoke this API on Relay Server to relinquish their ownership of the mailbox. Receiver device needs to present the currently established Receiver deviceClaim in order to relinquish their ownership of the mailbox. Once relinquished, the mailbox can be bound to a different Receiver device that presents its deviceClaim in a ReadSecureContentFromMailbox call.
+
+### Endpoint
+
+PATCH /{version}/m/{mailboxIdentifier}
+
+### Request Parameters
+
+Path parameters:
+
+- version (String, Required) - the version of the API. At the time of writing this document, “v1”.
+- mailboxIdentifier(String, Required) - MailboxIdentifier (refer to Terminology).
+
+Header parameters:
+
+- deviceAttestation (String, Optional) - optional remote OEM device proprietary attestation data.
```

Comment:
If this is optional then it doesn't think it provides any security benefit. A bad actor could just not send one up. @matthewcbyington what do you think?